recipes/cereal/all/test_package/CMakeLists.txt

```
@@ -7,5 +7,5 @@ conan_basic_setup(TARGETS)
 find_package(cereal CONFIG REQUIRED)
 
 add_executable(${PROJECT_NAME} test_package.cpp)
-target_link_libraries(${PROJECT_NAME} cereal)
+target_link_libraries(${PROJECT_NAME} cereal::cereal)
```

```suggestion
target_link_libraries(${PROJECT_NAME} cereal)
```
`cereal` is the "official" target right ?

---

https://github.com/conan-io/conan-center-index/pull/8979#pullrequestreview-856732018